tests/adapters/switches/remote_test.py

```diff
@@ -119,10 +151,10 @@ def test_start_transaction_fails_to_obtain_a_session(self, m_uuid):
                 status_code=500))
 
         with self.assertRaises(AnException):
-            self.switch.start_transaction()
+            self.switch.connect()
 
     @mock.patch('uuid.uuid4')
-    def test_end_session_fails_returns_to_normal_behavior(self, m_uuid):
+    def test_disconnect_fails_to_return_to_normal_behavior(self, m_uuid):
```

Comment:
disconnect fails returns to normal behavior
not
disconnect fails to return to normal behavior

as in, if disconnect fails the switch returns to a normal behavior where it can continue to do stuff